etcdmgmt/etcd-mgmt.go

```diff
@@ -6,6 +6,7 @@ import (
 	"net/http"
 	"os"
 	"os/exec"
+	"syscall"
```

Comment:
Don't use `syscall`. Use `golang.org/x/sys/unix` instead. Refer the NOTE in https://golang.org/pkg/syscall/